dbux-data/src/dataProviderUtil.js

```diff
@@ -272,44 +272,37 @@ export default {
       filter(staticContext => {
         return staticContext.displayName?.toLowerCase().includes(searchTerm);
       }).
-      map(staticContext =>
+      flatMap(staticContext =>
         dp.indexes.executionContexts.byStaticContext.get(staticContext.staticContextId)
-      ).
-      flat();
+      );
   },
 
-  findContextsByTraceSearchTerm(dp, searchTerm) {
+  searchTraces(dp, searchTerm) {
```

Comment:
A bit of clean up - can you:

* move all search-related functions together
* rename `find` to `search`
* add a `comment-barrier-1`: "search" to mark the whole region

?